README.md

```diff
@@ -47,6 +47,31 @@ OIDC_CLIENT_CONFIG = {
 }
 ```
 
+#### Gain EBoard Privileges Locally 
+In order to view pages that normally only eboard members can see it is reccomended that you do this these things:
+
+* Set up a local database.
+
+To do this through docker, run 
+
+```
+docker run --name postgres --mount target=/var/lib/postgresql/data,type=volume,src=conditional-db -e POSTGRES_PASSWORD=mysecretpassword -d -p 5432:5432 postgres
```

Comment:
```suggestion
docker run --name postgres -e POSTGRES_PASSWORD=mysecretpassword -d -p 5432:5432 postgres
```
You don't need to mount anything, the container db will persist as long as the container sticks around.

Reply:
Also, you should change config.env.py to use the connection string for this as the default.